pkgs/development/libraries/quarto/fix-deno-path.patch

```diff
@@ -0,0 +1,17 @@
+--- a/bin/quarto
++++ b/bin/quarto
+@@ -80,7 +80,7 @@ else
+   QUARTO_TARGET=${SCRIPT_PATH}/quarto.js
+   export QUARTO_BIN_PATH=$SCRIPT_PATH
+ 
+-  QUARTO_IMPORT_ARGMAP=--importmap=$SCRIPT_PATH/vendor/import_map.json
++  QUARTO_IMPORT_ARGMAP=--importmap=$SCRIPT_PATH/import_map.json
```

Comment:
We should move that file to ``$out/share/quarto/import_map.json``.

Reply:
The structure is related to the original structure of the installation archive. Why do you want to change it?

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/186697/commits/2290f013a440c8bad7165e63cbd56ecb6a2fb593